self-service/forgerock-selfservice-example-ui/pom.xml

```diff
@@ -13,70 +13,59 @@
 
     Copyright (c) 2015 ForgeRock AS. All Rights Reserved
-    Portions Copyright 2017-2023 Wren Security.
+    Portions Copyright 2017-2024 Wren Security.
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <artifactId>forgerock-selfservice</artifactId>
         <groupId>org.wrensecurity.commons</groupId>
+        <artifactId>forgerock-selfservice</artifactId>
         <version>22.7.0-SNAPSHOT</version>
     </parent>
 
     <artifactId>forgerock-selfservice-example-ui</artifactId>
-    <packaging>pom</packaging>
 
     <name>Wren Security Commons Self Service - Example UI</name>
     <description>UI for Wren Security Self-Service Example</description>
 
     <properties>
-        <forgerock-ui.version>22.1.2</forgerock-ui.version>
+        <wrensec-ui.version>23.1.0</wrensec-ui.version>
+        <node.install.directory>${project.basedir}</node.install.directory>
     </properties>
 
-    <build>
-        <resources>
-            <resource>
-                <directory>${basedir}/src/main/resources</directory>
-                <filtering>true</filtering>
-            </resource>
-
-            <resource>
-                <directory>${basedir}/src/main/js</directory>
-                <filtering>true</filtering>
-            </resource>
-        </resources>
-
-        <plugins>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>jslint-maven-plugin</artifactId>
-                <version>1.0.1</version>
-
-                <configuration>
-                    <excludes>
-                        <exclude>**/libs/*.js</exclude>
-                        <exclude>**/*Test.js</exclude>
-                    </excludes>
-                </configuration>
+    <dependencies>
+        <dependency>
+            <groupId>org.wrensecurity.commons</groupId>
+            <artifactId>wrensec-ui-commons</artifactId>
+            <version>${wrensec-ui.version}</version>
+            <type>zip</type>
+            <classifier>user</classifier>
+        </dependency>
 
-                <executions>
-                    <execution>
-                        <phase>validate</phase>
+        <dependency>
+            <groupId>org.wrensecurity.commons</groupId>
+            <artifactId>wrensec-ui-commons</artifactId>
+            <version>${wrensec-ui.version}</version>
+            <type>zip</type>
+            <classifier>build</classifier>
+        </dependency>
 
-                        <goals>
-                            <goal>jslint</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
+        <dependency>
+            <groupId>org.mozilla</groupId>
```

Comment:
This feels as unnecessary as it seems you have removed LESS compilation from pom.xml.

Reply:
Removed.